### PR TITLE
[feat] Update compiled circuit shape

### PR DESCRIPTION
**Pull request by ytham**
- Updates compiled circuit to include an optional `capacity` parameter

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "crypto"
   ],
   "dependencies": {
-    "@axiom-crypto/circuit": "2.0.5",
+    "@axiom-crypto/circuit": "link:../circuit/dist",
```
Comment (yi-sun) [inline client/package.json:27]: Is this intentional?

**Review by yi-sun — APPROVED** [review]
LGTM pending comment

**Review by jonathanpwang — APPROVED** [review]
LGTM:
I thought Roshan had added a warning console message when the capacity wasn't default to disclaim that the query would not be fulfilled onchain. I don't see it anymore (unless it is in sdk-core). Let's add one?

File: client/package.json (hunk 1 above)
Comment (ytham) [reply client/package.json:27]: yes, local links work in this repo now because the publish-all script will sanitize

**Comment by ytham** [comment]
@jonathanpwang added comment and test for higher capacity but the query still fulfilled with 256 (for both fields): https://explorer.axiom.xyz/v2/sepolia/query/60975000890293141607074507975632966094031070875632543324353343479256015819179

**Pull request merged**